tests/api/test_types.py

```diff
@@ -125,3 +125,16 @@ def test_sas_code():
     for invalid in ["", "AAA", "AAAAA", "aaaa", "AAAI", "AAAO", "AAA0", "AAA1"]:
         with pytest.raises(ValueError):
             SASCode(invalid)
+
+
+def test_entryname_normalization():
+    nfc_str = "àáâäæãåāçćčèéêëēėęîïíīįìłñńôöòóœøōõßśšûüùúūÿžźż"
+    nfd_str = (
+        "àáâäæãåāçćčèéêëēėęîïíīįìłñńôöòóœøōõßśšûüùúūÿžźż"
+    )
+
+    for i in range(len(nfc_str)):
+        assert nfc_str[i] != nfd_str[i]
```

Comment:
Well I guess my previous joking comment was of use in the end ^^
so we cannot access item by index this way for the nfd:
```python
>>> nfd_str = "àáâäæãåāçćčèéêëēėęîïíīįìłñńôöòóœøōõßśšûüùúūÿžźż"
>>> nfc_str = "àáâäæãåāçćčèéêëēėęîïíīįìłñńôöòóœøōõßśšûüùúūÿžźż"
>>> len(nfd_str)
89
>>> len(nfc_str)
47
>>> nfc_str[0]
'à'
>>> nfd_str[0]
'a'
>>> nfd_str[1]
'̀'
>>> nfd_str[88]
'̇'
>>> nfd_str[87]
'z'
>>> nfc_str[46]
'ż'
```

And we cannot use a index trick (i.g. `nfc_str[i] != nfd_str[2 *i] + nfd_str[2 * i + 1]`) given a) `2 * len(nfc_str) != len(nfd_str)` and b) there is no guarantee a nfd character is composed of only two codepoints

I think we should just do `assert nfd_str != nfc_str` (just to be sure an auto-formatter doesn't come and do nfc normalization on this source file 😄 )